compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -492,6 +543,21 @@ static LogicalResult setMatmulNoPadRootConfig(
       DispatchLoweringPassPipeline::CPUDoubleTilingExpert);
 }
 
+static LogicalResult setX86RootConfig(func::FuncOp entryPointFn,
+                                      linalg::ContractionOpInterface op,
+                                      ArrayRef<int64_t> flowTileSizes,
+                                      ArrayRef<int64_t> workgroupTileSizes,
+                                      int vectorSize) {
+  SmallVector<int64_t> l1TileSizes = {0, 0, 384};
```

Comment:
We should add a comment about why we are using these tile sizes and what is the plan to make this generic. I would strongly prefer if we could avoid adding model-specific hard-coded numbers.